### ISSUE TITLE
increase output size

### ISSUE THREAD

**Issue by hiogo9** [zhaoyuzhi/ChildPredictor#5]
Hi, thanks for great model! 
Suffered a little with dependencies, but launched it on 11 cuda and 2.0.1 pytorch.
All works great, but I have a question: is it possible to increase the size of the output image, or it was set during training and cannot be changed?

**Comment by zhaoyuzhi** [comment]
Yes, the output resolution is fixed during training and at inference.

Maybe an upsampler can be used like StableDiffusion and BSRGAN for post-processing ChildPredictor's output.
